prime-router/src/main/kotlin/history/azure/ReportFileFunction.kt

```diff
@@ -313,6 +317,12 @@ abstract class ReportFileFunction(
             fun extractShowFailed(query: Map<String, String>): Boolean {
                 return query["showfailed"]?.toBoolean() ?: false
             }
+
+            fun extractReceivingOrgSvcStatus(query: Map<String, String>): CustomerStatus? {
```

Comment:
Sorry, I missed this in the first PR, but is the intention to let through a request with an invalid CustomerStatus?  Currently, this returns a `400` and
```
{
    "error": "No enum constant gov.cdc.prime.router.CustomerStatus.INACTIV"
}
```

Reply:
This gets caught in the try/catch of the `getListByOrg` function. Is there a way to catch the error before that?